search_action_domain/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+import ast
+
+from odoo.tests import Form, TransactionCase, tagged
+
+
+@tagged("models")
+class TestModels(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.partner_obj = self.env["res.partner"]
```

Comment:
Remove, it's better to don't have models this way to avoid them to get attached to a specific user and environment. If you use `self.env["res.partner"]` instead, it will take current env and user

Reply:
Done!